observation_portal/sciapplications/serializers.py

```diff
@@ -107,14 +124,34 @@ class ScienceApplicationUserReviewSerializer(serializers.ModelSerializer):
     semester = serializers.SerializerMethodField()
     abstract = serializers.SerializerMethodField()
     pdf_url = serializers.SerializerMethodField()
+    application_id = serializers.SerializerMethodField()
+    science_category = serializers.SerializerMethodField()
+    mean_grade = serializers.SerializerMethodField()
```

Comment:
These don't all need to be `SerializerMethodFields` - you can use [SlugRelatedField](https://www.django-rest-framework.org/api-guide/relations/#slugrelatedfield) instead for simple foreign key field lookups that are one level deep. It would look like this:
```
mean_grade = serializers.SlugRelatedField(read_only=True, slug_field="mean_grade", source="science_application_review")